frontend/integration-tests/protractor.conf.ts

```diff
@@ -31,6 +31,7 @@ export const config: Config = {
   plugins: [failFast.init()],
   capabilities: {
     browserName: 'chrome',
+    acceptInsecureCerts : true,
```

Comment:
What certs does this use?  Could it be having the same SA crt changes under it issues?

Reply:
This is to bypass the browser warning for self-signed certificates.